source/uwp/Renderer/lib/InputValue.cpp

```diff
@@ -266,12 +268,24 @@ HRESULT NumberInputValue::IsValueValid(_Out_ boolean* isInputValid)
     // If there is a value, confirm that it's a number and within the min/max range
     if (currentValue.IsValid())
     {
-        int currentInt;
         try
         {
-            std::string currentValueStdString = HStringToUTF8(currentValue.Get());
-            currentInt = std::stoi(currentValueStdString);
-            isValid &= (currentInt < max) && (currentInt > min);
+            const std::string currentValueStdString = HStringToUTF8(currentValue.Get());
+            int currentInt = std::stoi(currentValueStdString);
+
+            if (max.Get())
+            {
+                int maxInt;
+                RETURN_IF_FAILED(max->get_Value(&maxInt));
+                isValid &= (currentInt < maxInt);
+            }
+
+            if (min.Get())
+            {
+                int minInt;
+                RETURN_IF_FAILED(min->get_Value(&minInt));
+                isValid &= (currentInt > minInt);
```

Comment:
is this correct? currentInt can be equal to minInt. The same applies to maxInt.

Reply:
Good catch, and not introduced with this change. @paulcam206, is this going to make it into 1.3? it might be good to try to get the fix for this in there. I can make a separate change for that if it would be easier.

---
In reply to: [463375533](https://github.com/microsoft/AdaptiveCards/pull/4446#discussion_r463375533) [](ancestors = 463375533)

Reply:
sent mail on logistics for fixing this 👍

Reply:
this is 1.3-only behavior (duh). fix inbound shortly.